src/utils/math-util.ts

```diff
@@ -0,0 +1,10 @@
+export function clamp(input: number, min: number, max: number): number {
```

Comment:
Is this used anywhere?

Reply:
I can remove this. It was used in an earlier iteration of the color sensor